app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/summary/SummaryProductFragment.java

```diff
@@ -207,9 +207,28 @@ public void refreshView(State state) {
             isLowBatteryMode = true;
         }
 
-        //checks if no category information has been entered for the product
-        if(product.getCategoriesTags()==null || product.getCategoriesTags().isEmpty()) {
-            showCategoryPrompt=true;
+        //checks the product states_tags to determine which prompt to be shown
+        List<String> statesTags = product.getStatesTags();
+        if (statesTags.contains(product.getLang()+":categories-to-be-completed")) {
```

Comment:
not quite sure why you're trying to change the language and put an English states.
If you query on world.openfoodfacts.org, the states will always be `en:categories-to-be-completed`
If you query on fr.openfoodfacts.org, the states will always be `fr:categories-a-completer`
So the easiest way would be to use the english version of states, which is the canonical one

Reply:
Right! I understand the role of language. Okay, I will fix it to english.

Reply:
perfect 👍